vllm/model_executor/layers/quantization/gptq_marlin.py

```diff
@@ -252,7 +168,6 @@ def create_weights(
         if self.quant_config.desc_act:
             # Act-order case
             assert self.quant_config.group_size != -1
```

Comment:
@alexm-neuralmagic Can you remind me why act_order does not work with channelwise?